Assets/Scripts/Player/PlayerController.cs

```diff
@@ -536,6 +537,7 @@ private void HandleDash()
             }
             else if (PlayerState is PlayerStateEnum.Dash)
             {
+                _shake.Shake(1f,2f,true,true); //TODO: change temp parameters for visual testing
```

Comment:
This causes an NPE in every scene except your test scene, maybe `_shake?.Shake`?